portfolio/src/main/webapp/script.js

```diff
@@ -36,3 +38,12 @@ function handleOnScroll() {
   var scrolled = (winScroll / height) * 100;
   document.getElementById("myBar").style.width = scrolled + "%";
 }
+
+/**
+ * Adds messages.
+ */
+function getMessages() {
+  fetch('/data').then(response => response.json()).then((message) => {
+    document.getElementById('message-container').innerText = message;
+  });
+}
```

Comment:
You can end the file with a newline here.

Reply:
Fixed!